fetch.sh

```diff
@@ -48,6 +47,12 @@ do
     rm -rf *.tar.xz *.tar.gz debian-binary
 done
 
+# clone wayland-protocols and remove unnecessary files and folders
+git clone --depth 1 https://github.com/wayland-project/wayland-protocols root/usr/share/wayland-protocols
+pushd root/usr/share/wayland-protocols
+ls -A | grep -Ev "stable|unstable" | xargs rm -rf
+popd
```

Comment:
Since we're not relying on apt packages for this anymore, lets:

1. Move all these lines into the top of `generate-wayland.sh`
2. Clone into `./wayland-protocols` instead of `root/usr/share/wayland-protocols`
3. Don't worry about removing files after cloning, we can just have `generate-wayland.sh` remove the entire repo after we're done generating.